kie-dmn-core/src/test/java/org/kie/dmn/core/FlightRebookingTest.java

```diff
@@ -218,5 +215,10 @@ private LocalDateTime date( String date ) {
     private BigDecimal number( Number n ) {
         return BigDecimal.valueOf( n.longValue() );
     }
+
+    private String formatMessages(List<DMNMessage> messages) {
```

Comment:
Ok, but I'll add it to some DMNTestUtils because I've noticed it might be more helpful this way when I create "draft" JUnit test classes.